env.js

```diff
@@ -42,3 +42,16 @@ augur.connect({ ethereumNode, augurNode }, (err, connectionInfo) => {
     });
   }
 });
+
+augur.augurNode.on('disconnect', function() {
```

Comment:
Strings should use double-quotes.  Does the linter not complain about this? :thinking:  I wonder if we're accidentally skipping a directory or something.

Reply:
This dir isn't linted. Fixed.